pkg/container/docker/critoflags.go

```diff
@@ -71,3 +71,16 @@ func generateMountBindings(mounts ...cri.Mount) []string {
 	}
 	return result
 }
+
+func generatePortMappings(portMappings ...cri.PortMapping) []string {
+	result := make([]string, 0, len(portMappings))
+	for _, pm := range portMappings {
+		publish := fmt.Sprintf("%d:%d", pm.HostPort, pm.ContainerPort)
+		if pm.ListenAddress != "" {
+			publish = fmt.Sprintf("%s:%s", pm.ListenAddress, publish)
```

Comment:
ListenAddress can be an IPv6 address and I think that we need to add square brackets in that case,  check if this approach works for you https://github.com/kubernetes-sigs/kind/blob/edbae5fdd1f1670eef0323018ccb18ced552fd29/pkg/cluster/nodes/create.go#L62

Reply:
Fixed! Thanks!